app/Views/Templates/components/forms/text-input.blade.php

```diff
@@ -38,6 +37,10 @@
         $width = 'max-w-xs';
         $sizeClass = $size ? 'input-'.$size : '';
 
+    } elseif ($variant === 'fullWidth') {
+        $style= 'input-bordered  input-sm';
+        $width = 'w-full';
+        $sizeClass = $scale ? 'input-'.$scale : '';
     } else {
```

Comment:
for text-input component, I added a new variant called "fullWidth" as using the default did not take the full width of the input as it had both "w-full" and "max-w-xs". Would this be ok?